### DISCUSSION TITLE
Anyone interested to participating in Technovacion21 Hackathon

### DISCUSSION THREAD

**Discussion by vikasganiga05** [EddieHubCommunity/support#1522] [General]
Hello Folks,
@Yuvrajhere and @vikasganiga05 are planning to participate in the Technovacion21 hackathon (https://technovacion.netlify.app/) which will start on 13/5/21 till 15/5/21. We want someone who is either good in React.js or Express/MongoDB backend or the entire MERN Stack. Anyone interested in taking part kindly contact @Yuvrajhere or @vikasganiga05 .
Thank You :pray:

**Comment by aminoxix** [comment]
🚀  Ya sure, what are you guys planning for hacks?

**Reply by Yuvrajhere** [reply]
hey Thanks for showing up, From where did you find the discussion link, Are you on EddieHub discord?

**Reply by Yuvrajhere** [reply]
I got your discord, I have messaged you there :)

**Reply by hdck007** [reply]
Still need someone??

**Reply by Yuvrajhere** [reply]
Thanks for showing up, We are not looking for a teammate now, but we would love to participate in future hackathons with you.
You can share your discord, I will send a friend request to you. 😊
